pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -2217,7 +2223,7 @@ func mergeTCPFilterChains(incoming []*listener.FilterChain, pluginParams *plugin
 			}
 		}
 
-		if !conflictFound {
+		if !conflictFound && !replacedFallthrough {
```

Comment:
I am confused by this, why do we not add it if we replaced fallthrough?

Reply:
If we have already replaced a wildcard matching fallthrough FC with the incoming one, we will not need to add the incoming one again.

Reply:
I just read the code wrong, sgtm